DSharpPlus/Entities/Channel/Message/DiscordMessageBuilder.cs

```diff
@@ -168,16 +183,35 @@ public DiscordMessageBuilder HasTTS(bool isTTS)
         }
 
         /// <summary>
-        /// Sets if the message will have an Embed.
+        /// Sets the embed for the current builder.
         /// </summary>
-        /// <param name="embed">The embed that should be sent.</param>
+        /// <param name="embed">The embed that should be set.</param>
         /// <returns>The current builder to be chained.</returns>
         public DiscordMessageBuilder WithEmbed(DiscordEmbed embed)
         {
             this.Embed = embed;
             return this;
         }
 
+        /// <summary>
+        /// Appends an embed to the current builder.
+        /// </summary>
+        /// <param name="embed">The embed that should be appended.</param>
+        /// <returns>The current builder to be chained.</returns>
+        public DiscordMessageBuilder AddEmbed(DiscordEmbed embed) => this.WithEmbed(embed);
```

Comment:
If you're changing `WithEmbed` this shouldn't point to that

Reply:
I'll fix that in a second